api/models/user-models.js

```diff
@@ -0,0 +1,9 @@
+const connection = require("../../db/connection");
+
+exports.fetchAllUsers = () => {
+  let queryString = `SELECT * FROM users`;
```

Comment:
You could just pop this in the connection.query, but if you do want to store it as a variable I wouldn't use let unless you plan on altering it!